src/api/EpiData.ts

```diff
@@ -142,18 +143,42 @@ export function loadDataSet(
   url.searchParams.set('format', 'json');
   return fetchImpl<Record<string, unknown>[]>(url)
     .then((res) => {
-      const data = loadEpidata(title, res, columns, columnRenamings, { _endpoint: endpoint, ...params });
-      if (data.datasets.length == 0) {
+      try {
+        const data = loadEpidata(title, res, columns, columnRenamings, { _endpoint: endpoint, ...params });
```

Comment:
can we put the `try` block around just this `loadEpidata()` line?  if the other lines are throwing exceptions, we should handle them in a different way.

Reply:
thats what im saying -- the `try` scope should contain ***only*** the line(s) where we anticipate a possible exception